redis/src/cluster_client.rs

```diff
@@ -147,11 +159,17 @@ impl ClusterClient {
             self.password.clone(),
         )
     }
+
+    /// Use `new()`.
+    #[deprecated(since = "0.22.0", note = "Use new()")]
+    pub fn open<T: IntoConnectionInfo>(initial_nodes: Vec<T>) -> RedisResult<ClusterClient> {
```

Comment:
Please keep this in the same place as the previous `open()` method at least for this commit, so it's obvious what changed.

Reply:
I've deliberately moved deprecated methods to the bottom of the struct to keep them out of the way.

Reply:
I'd prefer that that is done in a separate commit to faciliate review.